setversion.sh

```diff
@@ -42,6 +42,13 @@ update_package_json ()
     gsed -i "s/\"version\":.*\"[^\"]*\"/\"version\": \"${version}\"/g" ${file}
 }
 
+update_tools_postinstall_android_sh ()
+{
+    local file=$1
+    local version=$2
+    gsed -i "s/\(com.salesforce.mobilesdk:SalesforceHybrid:\)[0-9]\+.[0-9]\+.[0-9]\+/\1$version/g" ${file}
```

Comment:
@wmathurin - Here's an update to `setversion.sh` to update the MSDK version used in the template app's Gradle dependencies.